src/testhost.x86/TestHostTraceListener.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+namespace Microsoft.VisualStudio.TestPlatform.TestHost
+{
+#if !NET451
```

Comment:
Is `NET451` referring specifically to v4.5.1 of net fx or is this true for any 'desktop' net fx (newer than v4.5.1)? I am guessing its the latter but just wanted to check.

Reply:
Good point, I will use `NETCOREAPP` instead.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if